### ISSUE TITLE
NVDA is announcing the place holder information twice.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5018]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Visually challenged users who rely on screen readers are getting impacted if NVDA is announcing the place holder information twice.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the Url: [https://airtuserexpdev.azurewebsites.net/Dashboard](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fairtuserexpdev.azurewebsites.net%2fDashboard&data=05%7c01%7cv-shravyar%40microsoft.com%7c09aeb38517334354d48308dbeca14400%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638363949604189146%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=X9RebjeUY64a0z9YqF59KH8oRldy59wbtdE%2fzj0CjvM%3D&reserved=0) in Edge Browser->SAGE chat window will display.
Step 2: Type “What is the accessibility process”->Invoke “Dislike” button.
Step 3: Verify whether NVDA is announcing the place holder information twice or not.

### What do you expect?

NVDA should announce the place holder content only once.
Ensure: NVDA should announce as "Please let us know what information we could have provided to help you better edit multi line blank".

### What actually happened?

NVDA is announcing the place holder information twice.
Observation: NVDA is announcing as "Please let us know what information we could have provided to help you better edit multi line Please let us know what information we could have provided to help you better blank".

### Do you have any screenshots or recordings to repro the issue?

![MAS 1 3 1 - NVDA is announcing the place holder content twice](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/c845f82a-5756-4f41-aa15-1d03aa954f3c)

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645845](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645845)

**Comment by Kirank915** [comment]
#A11yusable;#A11ySev4;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
#A11yusable;#A11ySev4;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
This is related to Adaptive Cards. The input box is rendered by Adaptive Cards.

Please file a bug to their repo. If they think otherwise, please open a new bug and tag their response.
